extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/EmitterBuildItem.java

```diff
@@ -9,6 +9,10 @@
      */
     private final String name;
 
+    private String overflow;
```

Comment:
AFAIK build items should be immutable. What's the reason to make these fields mutable?